### ISSUE TITLE
Displayed extra whitespace when encode('MIME-Header', $subject) folds

### ISSUE THREAD

**Issue by memememomo** [dankogai/p5-encode#28]
My sample code is:

```
use strict;
use warnings;
use utf8;
use Encode;

my $subject = 'ああああああああああああ(あ)ああ';
print encode('MIME-Header', $subject);
```

The result is:

```
=?UTF-8?B?44GC44GC44GC44GC44GC44GC44GC44GC44GC44GC44GC44GC?=(
 =?UTF-8?B?44GC?=)=?UTF-8?B?44GC44GC?=
```

My mail client(Thunderbird) displays:

```
ああああああああああああ( あ)ああ
```

included whitespace after '('.

How do I fix it?

**Comment by khera** [comment]
This is a long debated bug the author doesn't seem to want to fix. The module is self-consistent in that when it decodes that header it eats the white space. The bug is more about the decoding error in the Encode::MIME::Header module than the encoding, but they are related.

https://rt.cpan.org/Ticket/Display.html?id=40027

I personally have been using MIME::EncWords to do the encoding, as Encode also encodes things it should not, such as email addresses, in violation of RFC 2047. It is a little more work to use it properly, but you do not get decoding errors in real mail clients.

**Comment by pali** [comment]
It is fixed in Encode 2.83.
